Homework6/ConsoleApp1/Task1.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Per requirements attributes and logger should be in a separate class library, so they should be in other project  then console app where they are been testing.